### ISSUE TITLE
Code for model prediction

### ISSUE THREAD

**Issue by sure7018** [Kevinz-code/CSRA#2]
Thank you for your excellent work. It has benefited me a lot。Is there any code about prediction？？

**Comment by Kevinz-code** [comment]
Hi,
Thanks for reading.

We provide validation code of various pretrained models on VOC and COCO, which could print out the mAP of each model (in ''val.py'').  Do you mean ''prediction'' by saying that the pretrained models could output a label set of each test image? Say,
image1: dog, person, cat.  
image2: person, bicycle.

If it was that case, you could use 0.5 as threshold to discriminate the positive and negative labels by (see line 45 in val.py):
result = nn.Sigmoid()(logit)
result[result >= 0.5] = 1
result[result < 0.5] = 0

Best,
Ke Zhu

**Comment by 2717117077** [comment]
Thank you for your reply。
By the way, it is estimated when the code of Vit and the code trained in the wide attribute dataset will be released, because my work is similar to that of the wide attribute dataset。
Thank you!

**Comment by Kevinz-code** [comment]
@2717117077 
Hi, actually I am a little busy these days.

Hopefully we are about to release them in the middle or late of September.

**Comment by sure7018** [comment]
Thank you and hope to see your excellent work as soon as possible！

**Comment by Kevinz-code** [comment]
Hi,
code concerned with wider attribute has been released @2717117077 
and the prediction model demo has also been added @sure7018 

Best

**Comment by sure7018** [comment]
Thank you. I have run through the code. I hope it can be helpful to the follow-up work, and I look forward to your excellent work in the future.

**Comment by abhigoku10** [comment]
@Kevinz-code  thanks for sharing the code base one query is for the WIDER attribute dataset will the output be person and its attributes or only person ??

**Comment by Kevinz-code** [comment]
Hi, @abhigoku10 

For Wider-Attribute dataset, every input image to the network is a person cropped from the original dataset. So the output will be the attributes (14 categories of attributes) of the certain person.

Best,

**Comment by abhigoku10** [comment]
@Kevinz-code  thanks for the response so just to clarify for Wider-Attribute dataset your not doing Detection + attribute classification but only attribute classification because the crop of the person is done offline . right ??

**Comment by Kevinz-code** [comment]
@abhigoku10 

"Wider-Attribute dataset your not doing Detection + attribute classification but only attribute classification because the crop of the person is done offline . right ??"
yes

This is a common fashion in multi-label benchmark.
